File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucChucNang.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucChucNang } from 'src/app/shared/DanhMucChucNang.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucChucNangService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucChucNang";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucChucNangService.cs
namespace Service.Implement
{
    public class DanhMucChucNangService : BaseService<DanhMucChucNang, IDanhMucChucNangRepository>
    , IDanhMucChucNangService
    {
    private readonly IDanhMucChucNangRepository _DanhMucChucNangRepository;
    public DanhMucChucNangService(IDanhMucChucNangRepository DanhMucChucNangRepository) : base(DanhMucChucNangRepository)
    {
    _DanhMucChucNangRepository = DanhMucChucNangRepository;
    }
    }
    }

